rails/app/javascript/components/Map.jsx

```diff
@@ -5,14 +5,14 @@ import MiniMap from "../vendor/mapboxgl-control-minimap";
 import Popup from "./Popup";
 
 // @NOTE: MAKE SURE ARRAY IS [LONGITUDE, LATITUDE]
-const defaultCenter = [-108, 38.5];
-const defaultBounds = [
-  [-180, -85], //southwest
-  [180, 85] //northeast
-];
-const defaultZoom = 3.5;
-const defaultPitch = 0;
-const defaultBearing = 0;
+// const defaultCenter = [-108, 38.5];
+// const defaultBounds = [
+//   [-180, -85], //southwest
+//   [180, 85] //northeast
```

Comment:
if w're not using this anymore, do we want to consider just pulling it out entirely?